docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/data-product-templates/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Data product templates"
+date: "2024-06-17"
+sidebar_label: "Using data product templates"
+sidebar_position: 2
+sidebar_custom_props:
+  offerings:
+    - bdp
+---
+
+## Creating Data Product based on Templates using Console
+
+To create a new data product based on a pre-defined template, navigate to the "Data products" section from the navigation bar and click the "Create data product" or "Templates" buttons.
+
+Below is a list of the current templates:
+
+### Base Web Data Product
+
+This Data Product template contains all the tracking of Standard Events. It is recommended to be used in conjunction with JavaScript Trackers (Web and Node.js).
+
+Note: The event volume counts are calculated differently for this Data Product. They are counts of any page_ping/page_view events sent for App Id's of this data product.
+
+Included event specifications:
+
+* [Page Ping](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/activity-page-pings/)
+* [Page View](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/page-views/)
+
+
+## E-commerce Web
+
+This Data Product template contains all the basic e-commerce interactions that help you analyze customer behavior, identify potential growth opportunities and improve your sales performance.
+
```

Comment:
We need to add:

Tracked using the [Snowplow Ecommerce plugin](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/ecommerce/).

Reply:
done thanks